src/Component/Best.jsx

```diff
@@ -1,14 +1,65 @@
-import React, {Component} from "react";
-
+import React, { Component } from "react";
+import axios from "axios";
+import moment from "moment";
+import CONSTANTS from "../Services/Hnapi";
+const { baseUrl } = CONSTANTS;
 export default class Best extends Component {
-    render() {
-        return (
+  constructor() {
+    super();
+  }
+  state = {
+    getBestItem: [],
+  };
+
+  componentDidMount() {
+    const promises = [];
+    axios
+      .get(`${baseUrl}/beststories.json`)
+      .then((result) => {
+        this.results = result.data.slice(0, 30);
+        this.results.forEach((element) => {
+          promises.push(
+            axios
+              .get(`${baseUrl}/item/${element}.json`)
+              .then((data) => data.data)
+          );
+        });
+        Promise.all(promises)
+          .then((stories) => {
+            this.setState({
+              getBestItem: stories.filter((s) => s),
+            });
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      })
+      .catch((err) => {
+        this.err = err;
```

Comment:
u can use react-toastify package for displaying error/success alerts

'https://www.npmjs.com/package/react-toastify'